### ISSUE TITLE
the position encoding equation is inconsistent with the equation in the paper

### ISSUE THREAD

**Issue by JaneShenYY** [facebook/MemNN#13]
Hi, thanks for sharing the code. Line9 - line 17 in MemN2N-babi-matlab/build_model.m implements the position encoding? It is different from the equation in the paper (section 4.1). Can you explain why they are not the same and why you chose to implement a different one here?

**Comment by tesatory** [comment]
yes you're right. there is some difference, but it is not intentional. although, the difference is mostly a constant factor, so it shouldn't make that much difference in the end.

**Comment by JaneShenYY** [comment]
@tesatory Thanks for the confirmation.
